scripts/makebumpver

```diff
@@ -413,8 +414,8 @@ class MakeBumpVer:
         f.write("%changelog\n")
         today = datetime.date.today()
         stamp = today.strftime("%a %b %d %Y")
-        f.write("* %s %s <%s> - %s-%s\n" % (stamp, self.gituser, self.gitemail,
-                                            newVersion, self.new_release))
+        f.write("* %s %s <%s> - %s%s\n" % (stamp, self.gituser, self.gitemail,
+                                           newVersion, self.new_release))
```

Comment:
This is not correct. You are removing `release` here, however, you need that in changelog. You need to add something like:
```
new_release = self.new_release or "-1"
```